icons/modrinth.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Modrinth</title><path d="M23.59 15.146a11.94 11.94 0 0 0 .003-6.238A11.989 11.989 0 0 0 8.866.419C3.895 1.742.518 5.953.002 10.784h2.03A9.963 9.963 0 0 1 9.35 2.389c4.971-1.332 10.084 1.36 11.866 6.097l-1.975.53a7.88 7.88 0 0 0-4.924-4.493l-.363 2.054a5.808 5.808 0 0 1 3.611 3.949 5.796 5.796 0 0 1-3.142 6.754l.54 2.016a7.87 7.87 0 0 0 4.783-8.301l1.97-.525a9.87 9.87 0 0 1-.181 3.949zm-8.488 8.442C8.676 25.304 2.11 21.505.375 15.098a12.05 12.05 0 0 1-.38-2.241h2.03a10.061 10.061 0 0 0 .915 3.358l1.811-1.088a7.96 7.96 0 0 1-.383-1.098c-1.126-4.202 1.37-8.535 5.58-9.661a7.905 7.905 0 0 1 2.383-.263l-.363 2.054a5.82 5.82 0 0 0-1.486.2c-3.105.83-4.972 4.019-4.118 7.128.051.191.111.377.18.558l2.317-1.388-.699-1.848 2.186-2.246 2.762-.596.797.985-1.275 1.29-1.112.35-.793.815.39 1.084.787.834 1.112-.296.792-.867 1.73-.549.517 1.159-1.782 2.185-2.988.947-1.341-1.491-2.336 1.402a5.847 5.847 0 0 0 4.878 1.965l.54 2.017a7.856 7.856 0 0 1-7.177-2.903l-1.8 1.079a9.963 9.963 0 0 0 10.505 3.588 9.929 9.929 0 0 0 6.378-5.253l1.998.727c-1.449 3.11-4.211 5.58-7.785 6.519z"/></svg>
```

Comment:
The favicon would be the better choice to use here. Please refer to #7579.

Reply:
Can you please tell me why the favicon would be better even if the documentation says that SVG should be considered? This optimized icon fulfills the outcomes. So, how is this a better choice?
As per documentation `Working with an SVG version of the logo is best`.
And it takes time to actually optimize the icon so I would request you to please merge the changes since there are no linting errors.

Reply:
The favicon can be found in SVG format in the art repository: https://github.com/modrinth/art/blob/main/Branding/Favicon/favicon.svg
Note that the favicon in the art repository is the future favicon, not the currently used one - it is better to future-proof, though.

The favicon makes more sense to use here as it is specifically designed for use in smaller contexts, the applications where Simple Icons are most common.

> so I would request you to please merge the changes

I am not a Simple Icons maintainer or contributor - I am, however, a representative of Modrinth.

Reply:
Okay, give me some time to make the changes for the pull request

Reply:
Thank you @wafflecoffee for your time and review. We always appreciate input from brand representative. Since the favicon is available as SVG format, it would be perfect to work from it.